app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] KIndly make sure you can create the user using the `sign_up` form as required for the project. Right now a **user** can be created but not registered in the database. If check the database once you `signed_up` you will find that the database is still empty.

- [ ] Also make sure when you have `signed_up` with a new_user, the created user is able to **Login**. Right now when you try to **login** the app raises an error as the database is empty. You can check again your **devise** configuration to make sure you did it correctly.

Reply:
done, but that was a problem with rails 7 and not the code